app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -454,6 +448,41 @@ fun PocketStoriesCategories(
     }
 }
 
+/**
+ * Wrapper for the color parameters of [PocketStoriesCategories].
+ *
+ * @param selectedTextColor Text [Color] when the category is selected.
+ * @param unselectedTextColor Text [Color] when the category is not selected.
+ * @param selectedBackgroundColor Background [Color] when the category is selected.
+ * @param unselectedBackgroundColor Background [Color] when the category is not selected.
+ */
+data class PocketStoriesCategoryColors(
+    val selectedTextColor: Color,
+    val unselectedTextColor: Color,
+    val selectedBackgroundColor: Color,
+    val unselectedBackgroundColor: Color,
+) {
+    companion object {
+
+        /**
+         * Builder function used to construct an instance of [PocketStoriesCategoryColors].
+         */
+        @Composable
+        fun buildColors(
+            selectedTextColor: Color = FirefoxTheme.colors.actionPrimary,
+            unselectedTextColor: Color = FirefoxTheme.colors.actionTertiary,
+            selectedBackgroundColor: Color = FirefoxTheme.colors.textActionPrimary,
+            unselectedBackgroundColor: Color = FirefoxTheme.colors.textActionTertiary,
+        ): PocketStoriesCategoryColors =
```

Comment:
```suggestion
        ) =
```
I think this might be unnecessary.